force-app/main/default/objects/Review__c/listViews/OpenReviews.listView-meta.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ListView xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>OpenReviews</fullName>
+    <columns>FundingRequest__c</columns>
+    <columns>NAME</columns>
+    <columns>Status__c</columns>
+    <columns>DueDate__c</columns>
+    <columns>Comments__c</columns>
+    <columns>AssignedTo__c</columns>
+    <filterScope>Everything</filterScope>
+    <filters>
+        <field>Status__c</field>
+        <operation>notEqual</operation>
+        <value>Completed</value>
+    </filters>
+    <label>Open Reviews</label>
```

Comment:
@nicolersmith This is the name of the new list view

![Screen Shot 2021-04-23 at 3 33 32 PM](https://user-images.githubusercontent.com/1134606/115931943-4610b500-a449-11eb-9871-14746f448ad5.png)